File: week1/김수빈/10814.py
import sys
input = sys.stdin.readline

n = int(input())
members = {}

for _ in range(n):
    age, name = input().split()
    members[int(age)] = [*members.get(int(age), []), name]

for age in sorted(members.keys()):
    for name in members[age]:
        sys.stdout.write(str(age) + " " + name + "\n")

File: week1/김고은/10814_나이순정렬/10814.js
let fs = require('fs');
let filePath = process.platform === "linux" ? "/dev/stdin" : __dirname + "/input.txt";
let input = fs.readFileSync(filePath).toString().trim().split("\n");

const [N, ...arr] = input;

let members = [];

for(let i =0; i < arr.length; i++) {
    const [age, name] = arr[i].split(' ')
    members.push([+age, name])
}

members.sort((a,b) => a[0] - b[0])

for(let i = 0; i < members.length; i++){
    console.log(members[i].join(' '))
}
